lib/phlex/sgml.rb

```diff
@@ -366,7 +366,7 @@ def yield_content_with_args(*args)
 
 			original_length = buffer.bytesize
 			content = yield(*args)
-			__text__(content) if original_length == buffer.bytesize
+			plain(content) if original_length == buffer.bytesize
```

Comment:
`plain` will raise if it’s passed something that doesn’t render, such as `nil`. But this happens all the time with implicit text. We should move the fragment guard from `plain` to `__text__` instead of switching to use `plain` instead of `__text__`.

Reply:
Done in https://github.com/phlex-ruby/phlex/pull/691/commits/f2e71930bdb058f679051d388cfb12c619135f4c.